servicex_app/servicex/templates/logs.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% block content %}
+
+<iframe
+    src="https://atlas-kibana.mwt2.org:5601/s/servicex/app/dashboards?auth_provider_hint=anonymous1#/view/bb682100-5558-11ed-afcf-d91dad577662?embed=true&_g=(filters%3A!(('%24state'%3A(store%3AglobalState)%2Cmeta%3A(alias%3A!n%2Cdisabled%3A!f%2Cindex%3A'923eaa00-45b9-11ed-afcf-d91dad577662'%2Ckey%3Ainstance%2Cnegate%3A!f%2Cparams%3A(query%3Aservicex)%2Ctype%3Aphrase)%2Cquery%3A(match_phrase%3A(instance%3A{{ config['INSTANCE_NAME'] }}))))%2CrefreshInterval%3A(pause%3A!t%2Cvalue%3A0)%2Ctime%3A(from%3Anow-24h%2Fh%2Cto%3Anow))&show-query-input=true&show-time-filter=true&hide-filter-bar=true">
```

Comment:
Let's move this to the app config so it can be a helm chart value. I'm sure other sites will have their own logging at some point